app/src/pages/inside/logsPage/modals/attachmentModal/attachmentImageModal.jsx

```diff
@@ -0,0 +1,72 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
+import { ModalLayout, withModal } from 'components/main/modal';
+import { messages } from './messages';
+import rotateImage from './img/rotate-left-icon.svg';
+import styles from './attachmentImageModal.scss';
+
+const cx = classNames.bind(styles);
+const classes = cx('toleft', null, {});
+
+@withModal('attachmentImageModal')
+@injectIntl
+export class AttachmentImageModal extends Component {
+  static propTypes = {
+    data: PropTypes.shape({
+      image: PropTypes.string,
+    }).isRequired,
+    intl: intlShape.isRequired,
+  };
+
+  state = {
+    rotationAmount: 0,
+  };
+
+  rotateImageHandler = () => {
+    const rotationA = this.state.rotationAmount;
+    this.setState({
+      rotationAmount: rotationA - 90,
+    });
+  };
+
+  generateRotationCommand = (amount) => `rotate(${amount}deg)`;
+
+  customButton = (intl) => (
+    <input
```

Comment:
We usually define icons in this way:
```
<div className={cx('rotate-icon'}) onClick={...}>
  {Parser(RotateImageIcon)}
</div>
```
Let's stick to the common way of doing this.

P.S. In the future, acessibility issues will be resolved by using aria-attributes, for now it's not necessary.